gcemetadata/lib/gcemetadata/gcemetautils.py

```diff
@@ -26,43 +26,46 @@
 
 from gcemetaExceptions import *
```

Comment:
Sorry, in context I don't see where "import *" does any harm in potentially confusing name spaces, thus not really seeing a need for a change.

Everything else is done